yt_dlp/extractor/yandextelemost.py

```diff
@@ -0,0 +1,132 @@
+
+from .common import InfoExtractor
+from ..utils import (
+    js_to_json,
+    url_or_none,
+)
+from ..utils.traversal import traverse_obj
+
+
+class YandexRuntimeStrmIE(InfoExtractor):
+    _VALID_URL = r'https?://runtime\.strm\.yandex\.ru/player/[a-z]+/(?P<id>[a-z0-9]+)'
+    _TESTS = [{
+        'url': 'https://runtime.strm.yandex.ru/player/episode/vple3e7omvkythqefn4a',
+        'info_dict': {
+            'id': 'vple3e7omvkythqefn4a',
+            'ext': 'mp4',
+            # match any timestamp appended for live streams
+            'title': r're:^ed3ab5fc53624ae19e4b65886bf4bb70.*',
+            'live_status': 'is_live',
+        },
+        'params': {'skip_download': True},
+        'only_matching': True,  # live stream ended
+    }, {
+        'url': 'https://runtime.strm.yandex.ru/player/episode/vpleontotb3a42ftvjod',
+        'info_dict': {
+            'id': 'vpleontotb3a42ftvjod',
+            'ext': 'mp4',
+            # match any timestamp appended for live streams
+            'title': r're:^64c006b42f044dd589723e6efeea332a.*',
+            'live_status': 'is_live',
+        },
+        'params': {'skip_download': True},
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        playlist = self._search_json(
+            r'window\.CloudVideo\.hydratePage\s*\(',
+            webpage, 'playlist', video_id, transform_source=js_to_json)
+        content = traverse_obj(playlist, ('firstItemData', 'content'))
```

Comment:
since we get nothing other than the `video_id` matched from the url without `content`, it's OK for it to be fatal
```suggestion
        content = self._search_json(
            r'window\.CloudVideo\.hydratePage\s*\(', webpage, 'playlist',
            video_id, transform_source=js_to_json)['firstItemData']['content']
```